src/BenchmarkDotNet/Toolchains/MonoWasm/WasmGenerator.cs

```diff
@@ -1,27 +1,82 @@
 ﻿using System.IO;
 using System.Text;
+using BenchmarkDotNet.Environments;
 using BenchmarkDotNet.Extensions;
 using BenchmarkDotNet.Helpers;
 using BenchmarkDotNet.Loggers;
 using BenchmarkDotNet.Running;
 using BenchmarkDotNet.Toolchains.CsProj;
 
+
 namespace BenchmarkDotNet.Toolchains.MonoWasm
 {
     public class WasmGenerator : CsProjGenerator
     {
         private readonly string CustomRuntimePack;
+        //TODO: Set this properly
+        private readonly bool Aot = true;
```

Comment:
Why is the default value `true` here while it's false everywhere else? Is the TODO still valid?

```suggestion
        //TODO: Set this properly
        private readonly bool Aot = true;
```